.travis.yml

```
@@ -10,9 +9,17 @@ matrix:
       sudo: required
   allow_failures:
     - python: "nightly"
-  
+
 # command to install dependencies
-install: 
-  - pip install pytest mock
+install:
+  - |
+    pip install kodi-addon-checker
+    pip install pytest mock
+
+before_script: rm -rf LICENSES/
+
 # command to run tests
-script: py.test -v
+script:
+  - |
+    py.test -v
+    kodi-addon-checker $TRAVIS_BUILD_DIR/service.notrobro/ --branch=leia
```

No need to make this a single 'command'.
```suggestion
script:
  - py.test -v
  - kodi-addon-checker --branch=leia -- service.notrobro/
```